frontend/app/services/cardService.ts

```diff
@@ -46,3 +47,27 @@ export const fetchCards = async (
 
   return await response.json();
 };
+
+/**
+ * Generates smileys for a given idiom text
+ * @param text - The idiom text to generate smileys for
+ * @returns Promise<string[]> - Array of emoji strings
+ */
+export const generateSmileys = async (text: string): Promise<string[]> => {
+  const url = new URL(API_ROUTES.SMILEYS, IDIOMS_BACKEND_URL);
+
+  const response = await fetch(url.toString(), {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+      'ngrok-skip-browser-warning': 'idioms',
+    },
+    body: JSON.stringify({ text }),
+  });
+
+  if (!response.ok) return handleApiError(response);
+
+  const result = await response.json();
+  return result.depiction || [];
+};
```

Comment:
We don't need this, the smileys are already coming as part of the current payload in the get call, you only need to extend the card type with the new fields and use them in the front-end. You can see the localhost:8000/docs for the new interface/types of the idioms now